README-cdk.md

```diff
@@ -224,17 +224,31 @@ oc create clusterrolebinding federation-admin \
           --serviceaccount="federation-system:default"
 ~~~
 
-Deploy the [cluster registry](https://github.com/kubernetes/cluster-registry) and the namespace where clusters are registered
-(`kube-multicluster-public`):
+Clone the Federation V2 repo and checkout the tag 0.0.4:
 
 ~~~sh
-oc create -f cluster-registry.yaml
+cd federation-v2/
+git checkout tags/v0.0.4
```

Comment:
I think you can do this on your repo and `git add` the `federation-v2` directory to the commit, and it should keep the reference you used (`v0.0.4`).
With that, and the `--recurse-submodules` clone option that you already added, I think this part of the readme (cloning federation-v2) can be removed, or replaced by a note that explains that the `federation-v2` repo has been checked out too.